ra2ce/analyses/indirect/traffic_analysis/traffic_analysis_base.py

```diff
@@ -0,0 +1,159 @@
+"""
+                    GNU GENERAL PUBLIC LICENSE
+                      Version 3, 29 June 2007
+
+    Risk Assessment and Adaptation for Critical Infrastructure (RA2CE).
+    Copyright (C) 2023 Stichting Deltares
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from abc import abstractmethod
+import abc
+import itertools
+import operator
+from typing import Any
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+import ast
+from ra2ce.analyses.indirect.traffic_analysis.accumulated_traffic_dataclass import (
+    AccumulatedTraffic,
+)
+
+
+class TrafficAnalysisBase(abc.ABC):
+    gdf: gpd.GeoDataFrame
+    od_table: gpd.GeoDataFrame
+    destinations_names: str
+
+    def optimal_route_od_link(
```

Comment:
This function seems to calculate regular traffic. That makes sense to me. But I think it does not match with the simple traffic weight meant in the documentation.